pages/common/docsify.md

```diff
@@ -0,0 +1,16 @@
+# docsify
+
+> Initialize and serve markdown documentation.
+> More information: <https://cli.docsifyjs.org>.
+
+- Initialize new documentation in the current directory:
```

Comment:
Not entirely sure, but I think this sounds a little more complete. Same for the second example
```suggestion
- Initialize a new documentation in the current directory:
```

Reply:
Adopted.